python/cudf_polars/cudf_polars/experimental/dask_serialize.py

```diff
@@ -17,6 +17,13 @@
 __all__ = ["register"]
 
 
+def frames_to_gpumemoryview(frames):
+    return [
+        plc.gpumemoryview(f) if hasattr(f, "__cuda_array_interface__") else f
```

Comment:
Doesn't `plc.gpumemoryview` have the `__cuda_array_interface__` attribute itself?

Can we just do:

```python
return map(plc.gpumemoryview, frames)
```

I suppose we could also drop the `frames_to_gpumemoryview` function altogether (and call `map(plc.gpumemoryview, frames)` in `*.deserialize`), but it's nice to have a docstring/comment explaining the need for the `plc.gpumemoryview` call somewhere.

Reply:
No, we can't do that because there may be frames that are not CUDA and we don't want to attempt to convert them to `plc.gpumemoryview`.

Reply:
@pentschev what do you think about putting which frames are on the GPU in the `header`, so that we can avoid these `hasattr` checks here? I'm not especially familiar with the context here, so this might be a non-starter.

Reply:
A frame being CUDA doesn't necessarily mean it supports CUDA array interface, it could also support only DLPack, or neither. Therefore, I think it's safer to keep the check for `__cuda_array_interface__`, although this doesn't look the nicest this is the canonical way we check for its support, for example [in UCX-Py itself](https://github.com/rapidsai/ucx-py/blob/0b12b68a2a37f872a02e686245facf36ab506c80/ucp/_libs/arr.pyx#L85). Furthermore, making any changes would mean making changes to the UCX-Py module in Distributed, at this point I'd rather avoid making any non-critical changes there until we switch to `distributed-ucxx` as default.